src/pile.js

```diff
@@ -376,79 +467,91 @@ const createPile = ({ initialItem, render, id, pubSub, store }) => {
         rotation = getRandomArbitrary(-10, 10);
       }
       let num = 0;
-      newItemsById.forEach((item, index) => {
+      newItems.forEach(item => {
+        const sprite = item.sprite;
         num++;
         let paddingX;
         let paddingY;
-        if (!Number.isNaN(+item.tmpAbsX) && !Number.isNaN(+item.tmpAbsY)) {
-          paddingX = item.x + x;
```

Comment:
After having added `item.x` etc. as explained above, we should be able to just use `item.x` etc. here.